app/controlplane/api/workflowcontract/v1/policy.proto

```diff
@@ -0,0 +1,62 @@
+//
+// Copyright 2024 The Chainloop Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package workflowcontract.v1;
+
+import "buf/validate/validate.proto";
+import "workflowcontract/v1/crafting_schema.proto";
+
+option go_package = "github.com/chainloop-dev/chainloop/app/controlplane/api/workflowcontract/v1";
+
+message Policy {
+  string api_version = 1 [(buf.validate.field).string.const = "workflowcontract.chainloop.dev/v1"];
+  string kind = 2 [(buf.validate.field).string.const = "Policy"];
+
+  Metadata metadata = 3 [(buf.validate.field).required = true];
+  PolicySpec spec = 4 [(buf.validate.field).required = true];
+}
+
+message Metadata {
+  // the name of the policy
+  string name = 3 [(buf.validate.field) = {
+    cel: {
+      message: "must contain only lowercase letters, numbers, and hyphens.",
+      expression: "this.matches('^[a-z0-9]([-a-z0-9]*[a-z0-9])?$')",
+      id: "name.dns-1123",
+    },
+    required: true,
+  }];
+}
+
+message PolicySpec {
+  // path to a policy script. It might consist of a URI reference
+  string path = 1 [(buf.validate.field).required = true];
```

Comment:
we might want to do here `oneof source` that for now it can only be a path, tomorrow it could be embed string.